src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -633,6 +643,66 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 	MonoTypeEnum arg0_type = fsig->param_count > 0 ? get_underlying_type (fsig->params [0]) : MONO_TYPE_VOID;
 
 	switch (id) {
+	case SN_Add:
+	case SN_Multiply:
+	case SN_Subtract: {
+		int instc0 = -1;
+		if (arg0_type == MONO_TYPE_R4 || arg0_type == MONO_TYPE_R8) {
+			switch (id) {
+			case SN_Add:
+				instc0 = OP_FADD;
+				break;
+			case SN_Multiply:
+				instc0 = OP_FMUL;
+				break;
+			case SN_Subtract:
+				instc0 = OP_FSUB;
+				break;
+			case SN_Max:
+				instc0 = OP_FMAX;
+				break;
+			case SN_Min:
+				instc0 = OP_FMIN;
+				break;
```

Comment:
These two cases aren't reachable, are they? (same for the else branch)

Reply:
Good catch, forgot to delete this code path after implementing the other one.